### ISSUE TITLE
Crashes when plugging in scanner via USB on Android 12

### ISSUE THREAD

**Issue by sobels** [zebra-technologies/Scanner-SDK-for-Android#8]
You can reproduce this by plugging in a scanner for which the app doesn't already have permission. On my device it instantly crashes 100% of the time.

According to the stack trace and decompiled code, the problem is in the `USBManager.getAvailableScannersList()` method. If it does not already have permission to open the attached device, it constructs an intent to get permission for that device with `PendingIntent.getBroadcast(context, 0, new Intent(ACTION_USB_PERMISSION), 0)`.

While this has been a warning for a while now, [starting in Android 12](https://developer.android.com/about/versions/12/behavior-changes-12#pending-intent-mutability) this will crash every time - [callers now have to specify](https://developer.android.com/guide/components/intents-filters#DeclareMutabilityPendingIntent) either `PendingIntent.FLAG_MUTABLE` or `PendingIntent.FLAG_IMMUTABLE` when calling this method. In your case, you should probably replace this with `PendingIntent.getBroadcast(context, 0, new Intent(ACTION_USB_PERMISSION), PendingIntent.FLAG_MUTABLE)`.

**tl;dr Use `PendingIntent.getBroadcast(context, 0, new Intent(ACTION_USB_PERMISSION), PendingIntent.FLAG_MUTABLE)`**

**Comment by sobels** [comment]
This was caused by me using an outdated library version (specifically, the one located in [this repository](https://github.com/zebra-technologies/Scanner-SDK-for-Android/tree/master/android_scanner_sdk_demo_app/BarcodeScannerLibrary)). You might want to update that version to eliminate confusion, but anyway updating to the latest SDK fixed my problem.
